app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,7 @@
+class CartsController < ApplicationController
+  def update
+    session[:cart] = { params[:id] => 1 }
+    flash[:notice] = 'Added to Cart'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.